src/service/print/getdocxdata.js

```diff
@@ -551,6 +551,8 @@ function imageData(image) {
     return (bufferImage = image.replace("data:image/jpg;base64,", ""));
   } else if (image.includes("data:image/jpeg;base64")) {
     return (bufferImage = image.replace("data:image/jpeg;base64,", ""));
+  } else if (image.includes("data:application/octet-stream;base64")) {
```

Comment:
@manojSRawat The application/octet-stream type is shown for documents when mimetype is unknown ( I have seen this when I have opened file in spreadsheets and word docs) ? (Correct me if I am wrong).   So considering it as an image will be wrong.